src/audio/signalinfo.h

```diff
@@ -36,7 +36,6 @@ class SignalInfo final {
 
     constexpr bool isValid() const {
         return getChannelCount().isValid() &&
-                getSampleLayout() &&
```

Comment:
Unintended change?

Reply:
No. The optional member is never set in our use case. Then isValid() would always return false and the fix does not work. Took me some time to find out why ;)

Not entirely sure about the semantics, but isValid() should only be applied to the content of an optional, not the hull. A missing optional is by default valid, otherwise it would be *mandatory* and not *optional*.

I needed to think about this already ;) https://github.com/slowtec/semval/blob/fe181085a3c62574bdc74a6c39d42352ca413b85/src/lib.rs#L105

Reply:
TL;DR I consider this removal a bug fix